mlog/global_test.go

```diff
@@ -39,11 +39,11 @@ func TestLoggingAfterInitialized(t *testing.T) {
 				FileLevel:     mlog.LevelDebug,
 			},
 			[]string{
-				`{"level":"debug","ts":0,"caller":"mlog/global_test.go:0","msg":"real debug log"}`,
-				`{"level":"info","ts":0,"caller":"mlog/global_test.go:0","msg":"real info log"}`,
-				`{"level":"warn","ts":0,"caller":"mlog/global_test.go:0","msg":"real warning log"}`,
-				`{"level":"error","ts":0,"caller":"mlog/global_test.go:0","msg":"real error log"}`,
-				`{"level":"error","ts":0,"caller":"mlog/global_test.go:0","msg":"real critical log"}`,
+				`{"level":"debug","ts":0,"caller":"testing/testing.go:0","msg":"real debug log"}`,
```

Comment:
Shouldn't the caller be mlog/global_test and not testing/testing? Looks like something changed and the stack offsets may need to be adjusted. Fine with merging this but we should open a ticket to investigate.

Reply:
Looks like removing the `zap.AddCallerSkip(1)` from `mlog.NewLogger` passes the tests. @cpanato, can you revert these changes and drop that line instead?

Reply:
sure, i was not sure about that test :/

Reply:
PTAL